src/Node/FS/Async.purs

```diff
@@ -132,6 +134,15 @@ stat :: FilePath
 stat file cb = mkEffect $ \_ -> runFn2
   statImpl file (handleCallback $ cb <<< (<$>) Stats)
 
+-- | Gets file or symlink statistics. `lstat` is identical to `stat`, except
+-- | that if the `FilePath` is a symbolic link, then the link itself is stat-ed,
+-- | not the file that it refers to.
+lstat :: FilePath
+     -> Callback Stats
+     -> Effect Unit
+lstat file cb = mkEffect $ \_ -> runFn2
+  lstatImpl file (handleCallback $ cb <<< (<$>) Stats)
```

Comment:
```diff
-lstatImpl file (handleCallback $ cb <<< (<$>) Stats)
+lstatImpl file (handleCallback $ cb <<< map Stats)
```

Reply:
I agree with that, yeah. I didn't even notice it though, because for `lstat`, I just copied the `stat` definition and changed out the impl function. I've changed `(<$>)` to `map` in `stat` as well.